src/imagery/i.eodag/i.eodag.py

```diff
@@ -660,21 +909,45 @@ def main():
 
     dates_to_iso_format()
 
+    if options["print"]:
```

Comment:
Instead of many if-else you could use a dictionary where the print functions are the "values"...